lib/services/sku-pack-service.js

```diff
@@ -82,6 +90,35 @@ function skuPackServiceFactory(
         return promises;
     };
 
+
+    SkuPackService.prototype.unregisterPack = function(contents, skuid) {
+        var promises = [];
+        var self = this;
+        try {
+            var conf = JSON.parse(contents);
+            
+            if(conf.hasOwnProperty('httpStaticRoot')) {
+                if( skuid in self.skuHandlers ) {
+                    delete self.skuHandlers[skuid];
+                }                    
+            }
+
+            if(conf.hasOwnProperty('httpTemplateRoot')) {
+                var httpTemplateRoot = path.resolve('/', conf.httpTemplateRoot);
+                httpTemplateRoot = self.confRoot + '/' + skuid + httpTemplateRoot;
+                promises.push(fs.readdirAsync(httpTemplateRoot));
+            }
+        } catch(error) {
+            logger.warning('Unable to unregister sku configuration for ' + skuid);
+            throw error;
+        }
+        return Promise.map(promises, function(entries) {
+            return Promise.all(_.map(entries, function(entry) {
```

Comment:
Doesn't really matter, but you could also do `Promise.map` here like on the previous line.